avrogen/src/main/java/org/kaaproject/kaa/avro/avrogen/TypeConverter.java

```diff
@@ -74,6 +74,51 @@ public static String convertToCType(String namespace, Schema schema) {
         return cType;
     }
 
+    public static String convertToObjCType(Schema schema) {
+        String objCType = "";
+        switch (schema.getType()) {
+            case BOOLEAN:
+                objCType = "BOOL";
+                break;
+            case INT:
+                objCType = "int32_t";
+                break;
+            case LONG:
+                objCType = "int64_t";
+                break;
+            case FLOAT:
+                objCType = "float";
+                break;
+            case DOUBLE:
+                objCType = "double";
+                break;
+            case STRING:
+                objCType = "NSString *";
+                break;
+            case BYTES:
+            case FIXED:
+                objCType = "NSData *";
+                break;
+            case ARRAY:
+                objCType = "NSArray *";
+                break;
+            case UNION:
+                objCType = "KAAUnion *";
```

Comment:
According to Apple naming conventions all classes should have unique names across all projects/bundle you may include. To achieve this 3-letter uppercase prefixes should be used as namespace. Please, visit the link below :)

 https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Conventions/Conventions.html#//apple_ref/doc/uid/TP40011210-CH10-SW4